addons/storyshots/storyshots-core/src/frameworks/react/renderTree.ts

```diff
@@ -1,8 +1,11 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
 import reactTestRenderer from 'react-test-renderer';
 
 function getRenderedTree(story: any, context: any, { renderer, ...rendererOptions }: any) {
-  const storyElement = story.render();
+  const StoryFn = story.render;
+  const storyElement = React.createElement(StoryFn);
   const currentRenderer = renderer || reactTestRenderer.create;
   const tree = currentRenderer(storyElement, rendererOptions);
 
```

Comment:
Instead of returning the full tree, we could return the first child here?

Reply:
on line 12

Reply:
🧠

Reply:
@brandon-leapyear if you want to do that that'd be amazing! if not, that's fine; I can take it from here?.

Reply:
I did a thing. Not sure if it's the correct change (is there an API we expect from `currentRenderer`?), so please let me know if it should be different

Reply:
I'd assume it should be the first child, rather then all children?

Did you update the snapshots?

Reply:
Yeah, the snapshots are up to date. `.children()` doesn't seem like it returns an array, so I wasn't sure how to get the first child

Reply:
Can i get access to push to your remote/branch?

I have a fix